package com.iu.exception;

public class ExceptionMain {
	public static void main(String[] args) {
		System.out.println("메인 시작");
		
		Test test = new Test();
		
		
	//	test.t4();
		System.out.println("메인 종료");
	}

}
